tools/finalize.py

```diff
@@ -1,63 +0,0 @@
-"""
-Finalize the repo for a release. Invokes towncrier and bumpversion.
-"""
-
-__requires__ = ['bump2version', 'towncrier', 'jaraco.develop>=7.21']
-
-
-import subprocess
-import pathlib
-import re
-import sys
-
-from jaraco.develop import towncrier
-
-
-bump_version_command = [
-    sys.executable,
-    '-m',
-    'bumpversion',
-    towncrier.release_kind(),
-]
-
-
-def get_version():
-    cmd = bump_version_command + ['--dry-run', '--verbose']
-    out = subprocess.check_output(cmd, text=True, encoding='utf-8')
-    return re.search('^new_version=(.*)', out, re.MULTILINE).group(1)
-
-
-def update_changelog():
-    towncrier.run('build', '--yes')
-    _repair_changelog()
-
-
-def _repair_changelog():
```

Comment:
Does excluding this file also mean that we no longer need to repair the changelog and that #2666 is already solved?

Reply:
Maybe so. I don't encounter the issue in other skeleton projects, so I assume so. If it turns out it's still a problem, I'll just strip out whatever customizations setuptools is using that introduce the bug. No reason to carry around this debt.